### DISCUSSION TITLE
Idea Pool

### DISCUSSION THREAD

**Discussion by Jacinto-S** [B-Team-Organisation/Fantasy-Chess#7] [Ideas]
Share ideas for potential future features for discussion.

**Comment by casqan** [comment]
# Ui Skins

https://github.com/czyzby/gdx-skins

**Comment by Albanovukelaj17** [comment]
# Tile Assets 

example:https://stock.adobe.com/de/845205200?clickref=1101lzS3JGTv&mv=affiliate&mv2=pz&as_camptype=search-sponsored&as_channel=affiliate&as_source=partnerize&as_campaign=cheezy&asset_id=845205200

idea : _TileGlas, _TileSand , Asesprite for self-creating Tiles

**Comment by adbennis** [comment]
# Increase Pixel Size

- For after MVP
